tests/e2e/ctl_v3_defrag_test.go

```diff
@@ -1,69 +0,0 @@
-// Copyright 2016 The etcd Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package e2e
-
-import (
-	"testing"
-
-	"go.etcd.io/etcd/tests/v3/framework/e2e"
-)
-
-func TestCtlV3DefragOnline(t *testing.T) { testCtl(t, defragOnlineTest) }
-
-func TestCtlV3DefragOfflineEtcdutl(t *testing.T) {
```

Comment:
I don't think we should migrate `TestCtlV3DefragOfflineEtcdutl` to common framework. It tests specifically `etcdutl` binary, that is not used in integration tests. You might have noticed that current framework doesn't really handle it well, which can be seen it additional changes were required to make it work. 

In the first pass of the migration we should migrate only tests for features available in API, as they should work exactly the same for etcd client used in integration tests and binaries used in e2e binaries.